### PR TITLE
Initial webpage setup

### PR DESCRIPTION
**Pull request by dthkao**
Still missing docs and FAQ.
Issue #29

### CODE REVIEW

**Comment by dthkao** [comment]
github/pages check is failing because it is assuming that configs are at the project root. We'll need to change settings to point to master branch docs/ after this gets merged.

**Comment by xpconanfan** [comment]
Can we add a readme for this subdir briefly describing how the pages are built?

**Comment by xpconanfan** [comment]
Let's get this baseline in so we can iterate upon it 👍

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,169 @@
+GEM
```
Comment (adorokhine) [inline docs/Gemfile.lock]: Is this intended for commit? Seems like an automatically generated file of some sort.

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: Mobly
+description: > # this means to ignore newlines until the next key
+  This is the official page for Mobly, a framework for running multiple-device tests.
+baseurl: "/mobly" # the subpath of your site, e.g. /blog
+url: "https://google.github.io"
```
Comment (adorokhine) [inline docs/_config.yml]: google.github.io/mobly seems to take us to a readme-only view. Is that intended?

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+permalink: /index.html
+permalink: /index.html
+is_site_nav_category: false
+site_nav_category_order: 1
+---
+
+Mobly is an extensive framework that supports test cases requiring multiple 
+devices, complex environments, or custom hardware setups.
+
+Mobly aslo provides a set of libraries to control and support testing of a
```
Comment (adorokhine) [inline docs/index.md]: Spelling: "aslo" (not sure if we're reviewing the content yet)

File: docs/Gemfile.lock (hunk 1 above)
Comment (dthkao) [inline docs/Gemfile.lock]: Nope, thanks.

File: docs/docs/example.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (adorokhine) [inline docs/docs/example.md]: Is this file needed?

File: docs/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+site_nav_category: docs
+is_site_nav_category: true
+site_nav_category_order: 10
+---
+{::options toc_levels="2"/}
+
+* TOC
+{:toc}
+
+## Android devices
```
Comment (adorokhine) [inline docs/docs/index.md]: This is just stub docs that you will expand soon right?

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (adorokhine) [inline docs/index.md]: What's the difference between this file and docs/docs/index?
